tests/beaker_tests/cisco_fabricpath_topology/test_fabricpath_topology.rb

```diff
@@ -0,0 +1,185 @@
+# Bash shell commands.
+#
+# Vegas and Bash Shell Commands:
+# 0   - successful command execution
+# > 0 - failed command execution.
+#
+# Puppet Commands:
+# 0 - no changes have occurred
+# 1 - errors have occurred,
+# 2 - changes have occurred
+# 4 - failures have occurred and
+# 6 - changes and failures have occurred.
+#
+# NOTE: 0 is the default exit_code checked in Beaker::DSL::Helpers::on() method.
+#
+# The test cases use RegExp pattern matching on stdout or output IO
+# instance attributes to verify resource properties.
+#
+###############################################################################
+
+require File.expand_path('../../lib/utilitylib.rb', __FILE__)
+
+# -----------------------------
+# Common settings and variables
+# -----------------------------
+testheader = 'Resource cisco_fabricpath_topology'
+
+# The 'tests' hash is used to define all of the test data values and expected
+# results. It is also used to pass optional flags to the test methods when
+# necessary.
+
+# 'tests' hash
+# Top-level keys set by caller:
+# tests[:master] - the master object
+# tests[:agent] - the agent object
+# tests[:show_cmd] - the common show command to use for test_show_run
+#
+tests = {
+  master:   master,
+  agent:    agent,
+  platform: 'n(5|6|7)k',
+}
+
+# tests[id] keys set by caller and used by test_harness_common:
+#
+# tests[id] keys set by caller:
+# tests[id][:platform] - a regexp pattern to match against supported platforms.
+#                        This key overrides a tests[:platform] key
+# tests[id][:desc] - a string to use with logs & debugs
+# tests[id][:manifest] - the complete manifest, as used by test_harness_common
+# tests[id][:resource] - a hash of expected states, used by test_resource
+# tests[id][:resource_cmd] - 'puppet resource' command to use with test_resource
+# tests[id][:ensure] - (Optional) set to :present or :absent before calling
+# tests[id][:code] - (Optional) override the default exit code in some tests.
+#
+# These keys are local use only and not used by test_harness_common:
+#
+# tests[id][:manifest_props] - This is essentially a master list of properties
+#   that permits re-use of the properties for both :present and :absent testing
+#   without destroying the list
+# tests[id][:resource_props] - This is essentially a master hash of properties
+#   that permits re-use of the properties for both :present and :absent testing
+#   without destroying the hash
+# tests[id][:title_pattern] - (Optional) defines the manifest title.
+#   Can be used with :af for mixed title/af testing. If mixing, :af values will
+#   be merged with title values and override any duplicates. If omitted,
+#   :title_pattern will be set to 'id'.
+#
+tests['non_default_properties'] = {
+  title_pattern:  '10',
+  manifest_props: "
+    member_vlans       => '10-20,30-40,100',
+    topo_name          => 'Topo-1',
+  ",
+  resource_props: {
+    'member_vlans' => '10-20,30-40,100',
+    'topo_name'    => 'Topo-1',
+  },
```

Comment:
You don't really have a test that exercises `range_summary()`. 

But you could do that here by just setting this to something like:

```
  manifest_props: "
    member_vlans       => '10-19, 20, 30, 21', # Intentionally mixed to test range_summary
    topo_name          => 'Topo-1',
  ",
  resource_props: {
    'member_vlans' => '10-21,30',
    'topo_name'    => 'Topo-1',
  },
```